scipy/stats/__init__.py

```diff
@@ -265,76 +266,159 @@
+
+These tests are often used to assess whether there is a relationship (e.g.
+linear) between paired observations in multiple samples or among the
+coordinates of multivariate observations.
+
+.. autosummary::
+   :toctree: generated/
+
+   linregress
    pearsonr
    spearmanr
    pointbiserialr
    kendalltau
    weightedtau
    somersd
-   linregress
    siegelslopes
    theilslopes
+   page_trend_test
    multiscale_graphcorr
 
-Statistical tests
-=================
+These association tests and related functions are suited to working
+with data in the form of contingency tables.
+
+.. autosummary::
+   :toctree: generated/
+
+   chi2_contingency
+   fisher_exact
+   barnard_exact
+   boschloo_exact
+   contingency.crosstab
+   contingency.expected_freq
+   contingency.margins
+   contingency.association
+   contingency.relative_risk
+   contingency.odds_ratio
+
+Independent Sample Tests
+------------------------
+Independent sample tests are typically used to assess whether multiple samples
+were independently drawn from the same distribution or different distributions
+with a shared property (e.g. equal mean).
+
+Some tests are specifically for comparing two samples.
 
 .. autosummary::
    :toctree: generated/
 
-   ttest_1samp
-   ttest_ind
    ttest_ind_from_stats
-   ttest_rel
-   chisquare
-   cramervonmises
-   cramervonmises_2samp
-   power_divergence
-   kstest
-   ks_1samp
-   ks_2samp
-   epps_singleton_2samp
+   poisson_means_test
+   ttest_ind
    mannwhitneyu
-   tiecorrect
-   rankdata
-   ranksums
-   wilcoxon
-   kruskal
-   friedmanchisquare
    brunnermunzel
-   combine_pvalues
-   jarque_bera
-   page_trend_test
-   tukey_hsd
-   poisson_means_test
+   mood
+   ansari
+   cramervonmises_2samp
+   epps_singleton_2samp
+   ks_2samp
+
+Others are generalized to multiple samples.
 
 .. autosummary::
    :toctree: generated/
 
-   ansari
-   bartlett
-   levene
-   shapiro
-   anderson
-   anderson_ksamp
-   binom_test
-   binomtest
+   f_oneway
+   tukey_hsd
+   kruskal
+   alexandergovern
    fligner
+   levene
+   bartlett
    median_test
-   mood
-   skewtest
-   kurtosistest
-   normaltest
-   goodness_of_fit
+   friedmanchisquare
+   anderson_ksamp
+
+Resampling Methods
+------------------
+The following functions can reproduce the p-value and confidence interval
+results of most of the functions above, and often produce accurate results in a
+wider variety of conditions. They can also be used to perform hypothesis tests
+and generate confidence intervals for custom statistics. This flexibility comes
+at the cost of greater computational requirements and stochastic results.
 
+.. autosummary::
+   :toctree: generated/
+
+   monte_carlo_test
+   permutation_test
+   bootstrap
+
+Other Test Functions
+--------------------
+The following functions are related to the tests above but do not belong in the
+above categories.
+
+.. autosummary::
+   :toctree: generated/
+
+   kstest
+   combine_pvalues
```

Comment:
Once we have `fdr_control`, we can include it with `combine_pvalues` in a Multiple Hypothesis/ Meta-analysis section.